Script_Source/Server/ICG_ansible_deployment/README.md

```diff
@@ -0,0 +1,29 @@
+# Linux Scripts
```

Comment:
ICG_ansible_deployment looks good -- I can roll this one in once my comments on Device are addressed.